flipperdevices/flipperzero-firmware
# NFC user dict list, delete, and de-duplication.

# What's new

- Mifare Classic user keys can be listed and deleted.
- Duplicate keys (from user DB) can't be added to user database.
- A few minor changes to icon names.

# Verification 

- Open Mifare Classic Keys menu in NFC
- If user dict is not empty, list button (right) will be active.
- Add a few keys, then select one in the list to delete.
- Verify the correct key was deleted.
- Try to add a key already present in user database, and get informed it already exists.

# Checklist (For Reviewer)
